blocks/note/note.css

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+.note {
+  border: 2px solid #707070;
+  border-radius: 4px;
+  color: #707070;
+  margin: 20px auto 0 auto;
+  display: table;
+  font-size: 14px;
+  width: 100%;
+  max-width: var(--grid-container-width);
+  position: relative;
+}
+
+.note .content div {
+  margin: 10px 45px 10px 20px;
+}
+
+.note > div.content:before {
+  color: black;
+  position: relative;
+  font-weight: bold;
+  content: 'Note:';
+  left: 20px;
+  top: 10px;
+}
+
+.note .icon {
+  position: absolute;
+  margin-top: 10px;
+  margin-right: 10px;
+  background: url(data:image/svg+xml,%3Csvg%20id%3D%22Layer_1%22%20data-name%3D%22Layer%201%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2018%2018%22%3E%3Cdefs%3E%3Cstyle%3E.cls-1%7Bfill%3A%23707070%3B%7D%3C%2Fstyle%3E%3C%2Fdefs%3E%3Ctitle%3EArtboard%205%3C%2Ftitle%3E%3Cpath%20id%3D%22_Compound_Path_%22%20data-name%3D%22%26lt%3BCompound%20Path%26gt%3B%22%20class%3D%22cls-1%22%20d%3D%22M9%2C2.12a7%2C7%2C0%2C1%2C0%2C7%2C7A7%2C7%2C0%2C0%2C0%2C9%2C2.12ZM8%2C5.62a.5.5%2C0%2C0%2C1%2C.5-.5h1a.5.5%2C0%2C0%2C1%2C.5.5v1a.5.5%2C0%2C0%2C1-.5.5h-1a.5.5%2C0%2C0%2C1-.5-.5Zm2.42%2C7.5H7.63s-.58%2C0-.58-.5.58-.5.58-.5H8v-3H7.6S7%2C9.11%2C7%2C8.62s.58-.5.58-.5H9.8a.2.2%2C0%2C0%2C1%2C.2.2v3.8h.42s.58%2C0%2C.58.5S10.42%2C13.12%2C10.42%2C13.12Z%22%2F%3E%3C%2Fsvg%3E) no-repeat;
+  background-size: 22px;
+  content: ' ';
+  height: 22px;
+  width: 22px;
+  top: 0px;
+  right: 0px;
+}
+
+/* caution */
+.note.caution {
+  border: 2px solid #e1951d;
```

Comment:
Could we please make the colour as CSS variable please

Reply:
done in [0f40da4](https://github.com/adobecom/helpx-internal/pull/3/commits/0f40da49ae8316b958698d5db6f0ffde24f496ae)